aten/src/ATen/native/README.md

```diff
@@ -112,6 +112,12 @@ argument name you provide, and a rename of an argument name is considered a BC-b
 change (e.g., you will probably need to update `tools/autograd/derivatives.yaml` at
 least). For more details please see the section on `variants`.
 
+As a convention we use 'out' to indicate an output argument. This aligns with the
```

Comment:
nit: this message is a little strange.  "This aligns with the Python bindings" is true, but the python bindings should be generated based on the schema here (and I think most would expect it is).

This also doesn't apply to operators with multiple output arguments.

"Check the generated code when making a change
to make sure you're not breaking the API when renaming an argument name of an
existing function."

this applies to any parameter you rename, not just output arguments.